test/perf/ParseBenchmarks.py

```diff
@@ -0,0 +1,264 @@
+# SPDX-License-Identifier: Apache-2.0
+
+######################################################################################################
+# There are four possible flags to add to this script:
+#
+# --run <Op>: Compute performance benchmarks for specified Op, and write to output file
+#
+# --runall: Compute performance benchmarks for all Ops in OpsWithPerformanceBenchmarks array
+#
+# --readrun <File> <Op>: Compute performance benchmarks for specified Op, and compare with benchmarks
+# already written to specified file (File must contain the same Op)
+#
+# --compare <Op1> <Op2>: Compare performance benchmarks written to file for both specified files (each
+# should contain the same Op)
+######################################################################################################
+
+
+
+import os
+import subprocess
+import sys
+import time
+
+

```

Comment:
Can you add a `print_usage(error_message)` which print the custom error message, followed by the  usage for the script, where all the main options and their subsequent parameters are listed? Kind of what you have on line 2?

We typically use `getopt`, which is a lot easier to manage and does a lot of the checking for you.  You could see in `utils/Mlir2FileCheck.py` how it is done. But don't change your code if its working and it would take you a lot of time to change.

```
        opts, args = getopt.getopt(
            argv, "hdca:n:m:", ["help", "debug", "check", "args=", "names=", "model="])
```